modules/genomic_browser/ajax/genomic_file_upload.php

```diff
@@ -107,6 +114,46 @@ function validateRequest()
 }
 
 /**
+ * Sets $fileToUpload->full_path
+ *
+ * @param stdClass $fileToUpload The file to upload
+ *
+ * @return void
+ */
+function setFullPath(&$fileToUpload)
+{
+    $config           = NDB_Config::singleton();
+    $genomic_data_dir = rtrim($config->getSetting('GenomicDataPath'), '/')
+        . "/genomic_uploader/";
```

Comment:
@xlecours Adding To Document For Release because I don't think the genomic_uploads folder is created on by default. This means for new installs and existing projects this fodler will need to be manually created.

That is, unless, we add code to create this folder if it doesn't exist -- but I imagine that's a 17.2 improvement and is too late to add now.